test_programs/execution_success/regression/src/main.nr

```diff
@@ -130,4 +130,8 @@ fn main(x: [u8; 5], z: Field) {
     assert(result_0 == 1);
     let result_4 = bitshift_variable(4);
     assert(result_4 == 16);
+
+    // Issue 6609
+    assert(u % -13 == 0);
```

Comment:
I think it's worth documenting how we define the modulo operator in the case where the rhs is negative.

Reply:
I have updated the signed integer section of the documentation